### PR TITLE
feat: conventional commit statistics

### PR DESCRIPTION
**Pull request by nedbat**
Tools for collecting and plotting statistics about conventional commit compliance.

These were moved here and adapted from https://github.com/nedbat/commitstats/tree/704d850b7b047b47f30d8c526650138359c5f0ca

### CODE REVIEW

**Comment by nedbat** [comment]
(This PR also includes #189.)

File: edx_repo_tools/conventional_commits/README.rst [hunk 1]
```diff
@@ -0,0 +1,39 @@
+Commit Stats
+############
+
+This directory has code to collect and report statistics about conventional commit compliance.  After cloning a number of repos, you use these tools to collect data into a commits.db database, then query and graph the data.
+
+#. Create a Python 3.8 virtualenv, and a new directory into which to clone the repos.
+
+#. Install repo-tools (https://github.com/edx/repo-tools) into your virtualenv.
```
Comment (feanil) [inline edx_repo_tools/conventional_commits/README.rst:8]:
```suggestion
#. Install repo-tools (https://github.com/edx/repo-tools) into your virtualenv.

    $ pip install /path/to/repo-tools
```

File: edx_repo_tools/conventional_commits/commitstats.py [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+    return data
+
+def load_commits(db, repo_name):
+    """Load the commits from the current directory repo."""
+
+    SEP = "-=:=-=:=-=:=-=:=-=:=-=:=-=:=-"
+    GITLOG = f"git log --no-merges --format='format:date: %aI%nhash: %H%nauth: %aE%nname: %aN%nsubj: %s%n%b%n{SEP}'"
+    SHORT_LINES = 5
+
+    # $ git log --format="format:---------------------%ndate: %aI%nhash: %H%nauth: %aE%nname: %aN%nsubj: %s%n%b"
```
Comment (feanil) [inline edx_repo_tools/conventional_commits/commitstats.py:49]: Looks like you changed the output format, can you update to show a sample of the new format or delete this comment?

**Review by feanil — APPROVED** [review]
Couple of suggestions but nothing blocking.

File: edx_repo_tools/conventional_commits/commitstats.py (hunk 2 above)
Comment (nedbat) [reply edx_repo_tools/conventional_commits/commitstats.py]: done.

File: edx_repo_tools/conventional_commits/README.rst (hunk 1 above)
Comment (nedbat) [reply edx_repo_tools/conventional_commits/README.rst]: done.

**Pull request merged**